src/YPS.Application/Parents/Queries/GetParentProfileInfo/GetParentProfileInfoQuery.cs

```diff
@@ -12,10 +12,10 @@
 
 namespace YPS.Application.Parents.Queries.GetParentProfileInfo
 {
-    public class GetParentsBySchoolQuety : IRequest<GetParentProfileInfoVm>
+    public class GetParentsProfileInfoQuery : IRequest<GetParentProfileInfoVm>
     {
         public long Id;
-        public class GetParentProfileInfoQueryHandler : IRequestHandler<GetParentsBySchoolQuety, GetParentProfileInfoVm>
+        public class GetParentProfileInfoQueryHandler : IRequestHandler<GetParentsProfileInfoQuery, GetParentProfileInfoVm>
```

Comment:
Set the name of the class to GetParentProfileInfoQuery, you're going to get the information of only one parent

Reply:
You did well with the handler name but not with the query